style.css

```diff
@@ -0,0 +1,35 @@
+#body{
+    background-image: url('https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg');
+    opacity: 0.8;
+    color: white;
+    }
+    #body-wrapper{
+        background-color:#1b1b32;
+        width: 50%;
+        margin: auto;
+    }
+    #main{
+        text-align: center;
+    }
+    #survey-form{
+        display: flex;
+        flex-direction:column;
+        max-width: 40%;
+        width: 100%;
+        margin: auto;
+    }
+    input{
+        height: 30px;
+    }
+    label{
+    margin-top: 10px;
+    }
```

Comment:
I think it would be better to use class attributes because It will select everything in the document with that class applied to it. kindly refer [curriculum-html-css](https://github.com/microverseinc/curriculum-html-css)